src/use-cases/commands/SupprimerUneNoteDeContexte.test.ts

```diff
@@ -0,0 +1,165 @@
+import { FindGouvernanceRepository, UpdateGouvernanceRepository } from './shared/GouvernanceRepository'
+import { FindUtilisateurRepository } from './shared/UtilisateurRepository'
+import { SupprimerUneNoteDeContexte } from './SupprimerUneNoteDeContexte'
+import { Gouvernance, GouvernanceUid } from '@/domain/Gouvernance'
+import { gouvernanceFactory, utilisateurFactory } from '@/domain/testHelper'
+import { Utilisateur, UtilisateurUid, UtilisateurUidState } from '@/domain/Utilisateur'
+import { epochTime } from '@/shared/testHelper'
+
+describe('supprimer une note de contexte d’une gouvernance', () => {
+  beforeEach(() => {
+    spiedGouvernanceUidToFind = null
+    spiedGouvernanceToUpdate = null
+    spiedUtilisateurUidToFind = null
+  })
+
+  it('étant donné une gouvernance existante, quand une note de context est supprimée par son gestionnaire, alors elle est supprimée', async () => {
```

Comment:
```suggestion
  it('étant donné une gouvernance existante, quand une note de contexte est supprimée par son gestionnaire, alors elle est supprimée', async () => {
```
Pareil pour tous les autres